library/parse_helm_repositories.py

```diff
@@ -0,0 +1,41 @@
+#!/bin/env python
```

Comment:
Additionally we should be using Python3

Reply:
The unit tests tun on what is in the container.

But the module is created to work with both versions:

```bash
docker run --rm -it \
  -v ${PWD}/:/tests/ \
  -w /tests \
  --entrypoint /bin/sh \
  python:3 -c "pip3 install ansible==${ANSIBLE_VERSION} && python3 /tests/library/test_parse_helm_repositories.py"
Collecting ansible==2.5
  Downloading https://files.pythonhosted.org/packages/4a/3b/9d98e132074bb6a3f18fd811db2819fbde6fc8a26fad9a40b49e53cb2455/ansible-2.5.0.tar.gz (10.1MB)
    100% |████████████████████████████████| 10.1MB 1.9MB/s 
Collecting jinja2 (from ansible==2.5)
  Downloading https://files.pythonhosted.org/packages/65/e0/eb35e762802015cab1ccee04e8a277b03f1d8e53da3ec3106882ec42558b/Jinja2-2.10.3-py2.py3-none-any.whl (125kB)
    100% |████████████████████████████████| 133kB 5.5MB/s 
Collecting PyYAML (from ansible==2.5)
  Downloading https://files.pythonhosted.org/packages/e3/e8/b3212641ee2718d556df0f23f78de8303f068fe29cdaa7a91018849582fe/PyYAML-5.1.2.tar.gz (265kB)
    100% |████████████████████████████████| 266kB 938kB/s 
Collecting paramiko (from ansible==2.5)
  Downloading https://files.pythonhosted.org/packages/4b/80/74dace9e48b0ef923633dfb5e48798f58a168e4734bca8ecfaf839ba051a/paramiko-2.6.0-py2.py3-none-any.whl (199kB)
    100% |████████████████████████████████| 204kB 4.8MB/s 
Collecting cryptography (from ansible==2.5)
  Downloading https://files.pythonhosted.org/packages/ca/9a/7cece52c46546e214e10811b36b2da52ce1ea7fa203203a629b8dfadad53/cryptography-2.8-cp34-abi3-manylinux2010_x86_64.whl (2.3MB)
    100% |████████████████████████████████| 2.3MB 2.0MB/s 
Requirement already satisfied: setuptools in /usr/local/lib/python3.7/site-packages (from ansible==2.5) (40.8.0)
Collecting MarkupSafe>=0.23 (from jinja2->ansible==2.5)
  Downloading https://files.pythonhosted.org/packages/98/7b/ff284bd8c80654e471b769062a9b43cc5d03e7a615048d96f4619df8d420/MarkupSafe-1.1.1-cp37-cp37m-manylinux1_x86_64.whl
Collecting pynacl>=1.0.1 (from paramiko->ansible==2.5)
  Downloading https://files.pythonhosted.org/packages/27/15/2cd0a203f318c2240b42cd9dd13c931ddd61067809fee3479f44f086103e/PyNaCl-1.3.0-cp34-abi3-manylinux1_x86_64.whl (759kB)
    100% |████████████████████████████████| 768kB 3.2MB/s 
Collecting bcrypt>=3.1.3 (from paramiko->ansible==2.5)
  Downloading https://files.pythonhosted.org/packages/8b/1d/82826443777dd4a624e38a08957b975e75df859b381ae302cfd7a30783ed/bcrypt-3.1.7-cp34-abi3-manylinux1_x86_64.whl (56kB)
    100% |████████████████████████████████| 61kB 25.8MB/s 
Collecting cffi!=1.11.3,>=1.8 (from cryptography->ansible==2.5)
  Downloading https://files.pythonhosted.org/packages/67/c7/b37e5f687e59499ea605ea3c0dccb0cc6b46bd96340a39fdad8e64b65bfb/cffi-1.13.1-cp37-cp37m-manylinux1_x86_64.whl (393kB)
    100% |████████████████████████████████| 399kB 8.7MB/s 
Collecting six>=1.4.1 (from cryptography->ansible==2.5)
  Downloading https://files.pythonhosted.org/packages/73/fb/00a976f728d0d1fecfe898238ce23f502a721c0ac0ecfedb80e0d88c64e9/six-1.12.0-py2.py3-none-any.whl
Collecting pycparser (from cffi!=1.11.3,>=1.8->cryptography->ansible==2.5)
  Downloading https://files.pythonhosted.org/packages/68/9e/49196946aee219aead1290e00d1e7fdeab8567783e83e1b9ab5585e6206a/pycparser-2.19.tar.gz (158kB)
    100% |████████████████████████████████| 163kB 14.8MB/s 
Building wheels for collected packages: ansible, PyYAML, pycparser
  Building wheel for ansible (setup.py) ... done
  Stored in directory: /root/.cache/pip/wheels/9f/48/10/6c5633adf264de3c5b49a66397a01cd5ee9653fa6155b34cf4
  Building wheel for PyYAML (setup.py) ... done
  Stored in directory: /root/.cache/pip/wheels/d9/45/dd/65f0b38450c47cf7e5312883deb97d065e030c5cca0a365030
  Building wheel for pycparser (setup.py) ... done
  Stored in directory: /root/.cache/pip/wheels/f2/9a/90/de94f8556265ddc9d9c8b271b0f63e57b26fb1d67a45564511
Successfully built ansible PyYAML pycparser
Installing collected packages: MarkupSafe, jinja2, PyYAML, pycparser, cffi, six, cryptography, pynacl, bcrypt, paramiko, ansible
Successfully installed MarkupSafe-1.1.1 PyYAML-5.1.2 ansible-2.5.0 bcrypt-3.1.7 cffi-1.13.1 cryptography-2.8 jinja2-2.10.3 paramiko-2.6.0 pycparser-2.19 pynacl-1.3.0 six-1.12.0
You are using pip version 19.0.1, however version 19.3.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
..
----------------------------------------------------------------------
Ran 2 tests in 0.000s

OK
```

@cytopia shall I add to `test-unit` target the following?

```bash
docker run --rm -it \
  -v ${PWD}/:/tests/ \
  -w /tests \
  --entrypoint /bin/sh \
  python:3 -c "pip3 install ansible==${ANSIBLE_VERSION} && python3 /tests/library/test_parse_helm_repositories.py"
```

This way we test both `python` versions. Though this slows down CI.

Reply:
> This way we test both python versions. Though this slows down CI.

Having both specifically is good I think. Jobs can run in parallel or?

Reply:
Testing now with Python 3.